### PR TITLE
add branded organization logo

### PR DESCRIPTION
**Pull request by Morantron**
How to test

* Open a rails console `bundle exec rails c`
* To show logo for organization with id 1, run `Redis.current.set('branded_organization_id', 1)`
* Reboot the app, in staging `sudo systemctl restart timeoverflow`

### CODE REVIEW

**Comment by enricostano** [comment]
@sseerrggii In which project this PR should be? 1T, 2T?

File: config/initializers/branded_organization.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+DEFAULT_BRANDED_ORG_ID = 246
+Rails.application.config.branded_organization_id = Rails.env.test? ? -1 : (Redis.current.get('branded_organization_id') || DEFAULT_BRANDED_ORG_ID)
```
Comment (enricostano) [inline config/initializers/branded_organization.rb]: Could you please add some documentation for that `-1` magic number?

Can't we just:
```ruby
unless Rails.env.test?
  Rails.application.config.branded_organization_id = (Redis.current.get('branded_organization_id') || DEFAULT_BRANDED_ORG_ID)
end
```

**Comment by sseerrggii** [comment]
> @sseerrggii In which project this PR should be? 1T, 2T?

1T

**Comment by sseerrggii** [comment]
Tested, works fine for me :+1: 

Waiting for a redeira approval

**Comment by sseerrggii** [comment]
a redeira approve :heavy_check_mark:

File: app/views/application/_brand_logo.html.erb [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<div class="container text-center" style="margin-bottom: 20px">
+  <%= image_tag("redeira@1x.png", class: 'organization-brand-logo', srcset: '/assets/redeira@2x.png 2x') %>
```
Comment (markets) [inline app/views/application/_brand_logo.html.erb]: are you sure the `srcset` works with compiled assets?

**Comment by markets** [comment]
I think it would be better to squash this PR 🙏 😛

**Comment by sseerrggii** [comment]
Wait before merge. A redeira team is thinking about the right logo

File: app/helpers/brand_logo_helper.rb [hunk 3]
```diff
@@ -7,6 +7,7 @@ def render_brand_logo
   private
 
   def should_render_logo?
+    byebug
```
Comment (enricostano) [inline app/helpers/brand_logo_helper.rb]: Elegant.

**Comment by sseerrggii** [comment]
Logo A Redeira Azul

This one @Morantron

File: app/helpers/brand_logo_helper.rb (hunk 3 above)
Comment (Morantron) [reply app/helpers/brand_logo_helper.rb]: :tophat: :ok_hand:

**Comment by Morantron** [comment]
new logo is on :dancer: 

image

it's as big as the provided image

**Comment by sseerrggii** [comment]
Perfect @Morantron 

Can we add "img-responsive" to a img class to avoid image overflow in small screens?

And... Why don't pass the test??  `The Travis CI build failed`  :thinking:

**Comment by Morantron** [comment]
pushing again with the new class, and rebased with latest `develop` ( this might fix travis thing )

**Comment by enricostano** [comment]
@Morantron @sseerrggii how it looks like with mobile app?

**Comment by Morantron** [comment]
haven't tried in the mobile app, but in small screens looks good :tm: 

image

**Comment by sseerrggii** [comment]
Tested, work fine, showed in 1 organization and don't in others

I tested also in app, I have this first screen and I was shocked :scream: , after that I used the app and everyhing works fine, when I reopened the app this first screen works fine... strange thing :thinking:  

 
Screenshot_20190314-141553

File: app/helpers/brand_logo_helper.rb [hunk 4]
```diff
@@ -0,0 +1,16 @@
+module BrandLogoHelper
+  def render_brand_logo
+    return unless should_render_logo?
+    render 'application/brand_logo'
+  end
+
+  private
+
+  def should_render_logo?
+    current_organization&.id == branded_organization_id
```
Comment (enricostano) [inline app/helpers/brand_logo_helper.rb]: Due to a bug in how we use the `current_organization_id` session key we need to make sure that we show the logo only in pages where the user is logged in.

```suggestion
    return false unless current_user
    current_organization&.id == branded_organization_id
```

**Comment by sseerrggii** [comment]
I created new issue for problems with app https://github.com/coopdevs/timeoverflow/issues/474

**Comment by sauloperez** [comment]
Not bad. Quite professional :ok_hand:

**Pull request merged**